lighthouse-core/test/test-utils.js

```diff
@@ -27,4 +28,16 @@ expect.extend({
 
     return {actual, message, pass};
   },
+
+  // Expose toBeCloseTo() so it can be used as an asymmetric matcher.
+  toBeApproximately(...args) {
```

Comment:
it's really dumb you can't use `toBeCloseTo` in `toMatchObject`, so this takes care of that :)